notebooks/05_Kubeflow_Pipeline/05_01_Pipeline_SDK.ipynb

```diff
@@ -66,7 +66,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!which dsl-compile"
+    "!~/.local/bin/dsl-compile"
```

Comment:
why do we use absolute path here?

Reply:
Since after we install `kfp`, the `dsl-compile` is located in the `~/.local/bin`. The command `which dsl-compile` will show nothing unless we do `export PATH=$PATH:~/.local/bin` in advance. 


**Image from** [kubeflow website](https://www.kubeflow.org/docs/pipelines/sdk/install-sdk/): 
![image](https://user-images.githubusercontent.com/23116624/77120242-940ceb00-69f5-11ea-9fbd-6640dbcd4f86.png)

Reply:
Can you try if you restart kernel? command installed by pip should be added to PATH automatically.

Reply:
I re-create a new notebook server with image `
gcr.io/kubeflow-images-public/tensorflow-1.15.2-notebook-cpu:1.0.0`.

After re-run this notebook, there's no output for `which dsl-compile`. Then I go to check the `~/.loca/bin` and found `dsl-compile` located there. 

The default `PATH` is `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`. 

Seems like pip haven't added it to the PATH underlying.

Reply:
Does it mean every time we need to use `~/.local/bin/dsl-compile`? Instead of doing that, I would suggest to `PATH="$HOME/.local/bin:$PATH"` at the beginning. I will file a PR to add `~/.local/bin` to container PATH

Reply:
Yes, we could do `PATH="$HOME/.local/bin:$PATH"`, this should works better.

Reply:
Since we'll fix the problem in [this issue](https://github.com/kubeflow/kubeflow/issues/4878), I'll revert this to the original version.